docs/source/get_hsup_bc_ic.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+# Command line arguments
+if [ -z "$1" ]; then
+   echo "Usage: $0 yyyymmddhh "
+   echo "For example: 2018100900, 2019052300, 2019071200, 2019092512, 2019102812, 2020011812, 2020020312, 2020040912, 2020072300" 
+   exit
+fi
+## date (year month day) and time (hour)
+yyyymmddhh=$1 #i.e. "2018100900"
+
+if [ "$#" -ge 2 ]; then
+  nfcst=$2 #i.e. "12"
+else
+  nfcst=90
+fi
+
+## forecast interval, the default interval are 3 hours
+if [ $yyyymmddhh -le 2019052300 ]; then
+   nfcst_int=6
+else 
+   nfcst_int=3
+fi
+
+# Get the data (do not need to edit anything after this point!)
+mkdir -p $yyyymmddhh
+echo "Download files for cycle $yyyymmddhh ..."
+cd $yyyymmddhh
+
+
+#getting online forecast data
```

Comment:
How do we prevent re-downloading files that have been downloaded successfully? Could we download the files to a temporary filename first, then check the return code from wget, and then move to the final name of the return code is zero?

Reply:
I can add a check to make sure it will not download the file if it is already downloaded.